gen

```diff
@@ -56,6 +56,31 @@ cat << EOF
 EOF
 }
 
+touchjs() {
+cat << EOF
+#!/usr/bin/env node
```

Comment:
Am I crazy or was there more code here for JS when I merged it? If not, I can move this into the touchsh function if all JS needs is a shebang...don't know anything about JS though.

Reply:
No, there wasn't. The larger one was for the HTML function.